README.md

```diff
@@ -57,6 +57,15 @@ open build/xcode/tangram.xcodeproj
 
 Note that any Xcode configuration change you make to the project won't be preserved when CMake runs again. Build configuration is defined only in the CMakeLists file(s).
 
+### OS X (CLion) ###
```

Comment:
Can you update this to also include `Linux`?, since CLion is available on Linux and works exactly how you described :D.
P.S. Do not have access permission to push to your folk of tangram repo.